### PR TITLE
NIFI-11004 Add documentation for OIDC groups claim property

### PR DESCRIPTION
**Pull request by exceptionfactory**
# Summary

[NIFI-11004](https://issues.apache.org/jira/browse/NIFI-11004) Updates the Administrator's Guide with documentation for the OIDC groups claim property and also updates the standard `nifi.properties` configuration with the default value of `groups`, matching the current internal behavior.

# Tracking

Please complete the following tracking steps prior to pull request creation.

### Issue Tracking

- [X] [Apache NiFi Jira](https://issues.apache.org/jira/browse/NIFI) issue created

### Pull Request Tracking

- [X] Pull Request title starts with Apache NiFi Jira issue number, such as `NIFI-00000`
- [X] Pull Request commit message starts with Apache NiFi Jira issue number, as such `NIFI-00000`

### Pull Request Formatting

- [X] Pull Request based on current revision of the `main` branch
- [X] Pull Request refers to a feature branch with one commit containing changes

# Verification

Please indicate the verification steps performed prior to pull request creation.

### Build

- [ ] Build completed using `mvn clean install -P contrib-check`
  - [ ] JDK 8
  - [ ] JDK 11
  - [ ] JDK 17

### Licensing

- [ ] New dependencies are compatible with the [Apache License 2.0](https://apache.org/licenses/LICENSE-2.0) according to the [License Policy](https://www.apache.org/legal/resolved.html)
- [ ] New dependencies are documented in applicable `LICENSE` and `NOTICE` files

### Documentation

- [X] Documentation formatting appears as expected in rendered files

### CODE REVIEW

File: nifi-docs/src/main/asciidoc/administration-guide.adoc [hunk 1]
```diff
@@ -503,6 +503,9 @@ JSON Web Key (JWK) provided through the jwks_uri in the metadata found at the di
 |`nifi.security.user.oidc.additional.scopes` | Comma separated scopes that are sent to OpenId Connect Provider in addition to `openid` and `email`.
 |`nifi.security.user.oidc.claim.identifying.user` | Claim that identifies the user to be logged in; default is `email`. May need to be requested via the `nifi.security.user.oidc.additional.scopes` before usage.
 |`nifi.security.user.oidc.fallback.claims.identifying.user` | Comma separated possible fallback claims used to identify the user in case `nifi.security.user.oidc.claim.identifying.user` claim is not present for the login user.
+|`nifi.security.user.oidc.claim.groups` | Name of the ID token claim that contains an array of group names of which the
+user is a member. Application groups must be supplied from a User Group Provider with matching names in order for the
+authorization process to use ID token claim groups. The default value is `groups`.
```
Comment (Lehel44) [inline nifi-docs/src/main/asciidoc/administration-guide.adoc:508]:
```suggestion
user is a member. Application groups must be supplied by a User Group Provider with matching names for the
authorization process to use ID token claim groups. The default value is `groups`.
```

**Review by Lehel44 — COMMENTED** [review]
Thanks for adding the documentation @exceptionfactory. Why do we need to update nifi-resources.pom?

**Comment by exceptionfactory** [comment]
> Why do we need to update nifi-resources.pom?

Thanks for the feedback @Lehel44. The updates to `pom.xml` in `nifi-resources` sets the default value for the `nifi.properties` configuration included in the standard distribution. This follows the convention of other default property values, and also allows custom builds to set different default values using Maven properties.

**Comment by Lehel44** [comment]
Thank you. LGTM+1

**Review by greyp9 — APPROVED** [review]
LGTM

**Pull request closed without merging**